projects/packages/my-jetpack/_inc/data/products/use-install-plugins.ts

```diff
@@ -0,0 +1,46 @@
+import { useGlobalNotices } from '@automattic/jetpack-components';
+import { __, sprintf } from '@wordpress/i18n';
+import { REST_API_SITE_PRODUCTS_ENDPOINT, QUERY_INSTALL_PRODUCT_KEY } from '../constants';
+import useSimpleMutation from '../use-simple-mutation';
+import useProducts from './use-products';
+
+const useInstallPlugins = ( productSlugs: string | string[] ) => {
+	const productIds = Array.isArray( productSlugs ) ? productSlugs : [ productSlugs ];
+
+	const { products, refetch } = useProducts( productIds );
+	const { createSuccessNotice } = useGlobalNotices();
+
+	const { mutate: install, isPending } = useSimpleMutation( {
+		name: QUERY_INSTALL_PRODUCT_KEY,
+		query: {
+			path: `${ REST_API_SITE_PRODUCTS_ENDPOINT }/install`,
+			method: 'POST',
+			data: { products: productIds },
+		},
+		options: {
+			onSuccess: () => {
+				refetch().then( () => {
+					createSuccessNotice(
+						sprintf(
+							/* translators: %s is the word "Plugin" or "Pluigns" (singular or plural). */
+							__( '%s installed successfully!', 'jetpack-my-jetpack' ),
+							products?.length === 1 ? products[ 0 ].title : __( 'Plugins', 'jetpack-my-jetpack' )
```

Comment:
Would it be possible to update the plural strings to account for localization? Instead of "'%s installed successfully!", we would prefer 2 strings or a proper plural:

* plugin installed successfully!
* plugins installed successfully!

In many languages, the words will need to change depending on whether the word "plugin" is singular or plural.

Reply:
Hey @emilyaudela 👋!
Thanks for your feedback!  Yes! Thank you! I'll get it fixed asap. 👍🙂

Reply:
Oh wait... yeah the translation comment is incorrect. The `%s` is not the word "Plugin" or "Plugins", but rather `%s` is the Jetpack product name, i.e.- VaultPress Backup or Boost or Social or Search or VideoPress, etc. if it's in the singular context, or `%s` is the word "Plugins" if it's in the plural context.
So yeah, anyway still, I fix the translation comment and make it 2 separate strings (or proper plural, whichever seems to be the best approach). 👍

Reply:
I've created PR https://github.com/Automattic/jetpack/pull/41756 which separated the above translation into separate strings.